crates/contracts/solidity/tests/Counter.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+contract Counter {
```

Comment:
Nit: all our other contracts have:

```suggestion
/// @title documentation
contract Counter {
```

To describe what the contract is used for. Can you add that here as well (I don't think it is needed for the functions/storage variables, its all self-explanatory). Mostly, it would be nice to include context of "what this is used for/by"